skimage/morphology/gray.py

```diff
@@ -255,20 +232,42 @@ def dilation(image, footprint=None, out=None, shift_x=False, shift_y=False):
         The array to store the result of the morphology. If None is
         passed, a new array will be allocated.
     shift_x, shift_y : bool, optional
-        Shift footprint about center point. This only affects 2D
-        eccentric footprints (i.e., footprints with even-numbered
-        sides).
+        Shift footprint about center point. This only affects 2D footprints
+        with even-numbered sides. These two parameters are deprecated in
+        favor of ``mirror``.
+
+        .. versionchanged:: 0.20
+            Parameters ``shift_x`` and ``shift_y`` are deprecated and will be
+            removed in a future version.
+
+    mirror : bool, optional
+        Mirror the footprint along each dimension. Default is ``False``.
+
+        .. versionadded:: 0.20
+            ``mirror`` was added in 0.20.
+
+    mode : {None, 'reflect','constant','nearest','mirror', 'wrap'}, optional
+        The ``mode`` parameter determines how the array borders are handled.
+        If ``None``, pixels outside the image domain are assumed to be the
+        minimum for the image's dtype, which causes them to not influence the
+        result. Default is ``'reflect'``.
+    cval : scalar, optional
+        Value to fill past edges of input if ``mode`` is 'constant'. Default
+        is 0.0.
+
+        .. versionadded:: 0.20
+            ``mode`` and ``cval`` were added in 0.20.
 
     Returns
     -------
-    dilated : uint8 array, same shape and type as `image`
+    dilated : uint8 array, same shape and type as ``image``
```

Comment:
Nitpick: 

```suggestion
    dilated : uint8 array, same shape and type as `image`
```
Currently, we have the convention to reference parameters this way. This also applies to a lot of other places in this PR.